### ISSUE TITLE
How to contribute

### ISSUE THREAD

**Issue by malvese** [dxdatabase/Open-source-film-database#3]
Great initiative, I'm a big fan of this database!

I wondered if it is possible to contribute film info, and how?

**Comment by Merinorus** [comment]
Hi,

You can open a pull request, like this:
https://github.com/dxdatabase/Open-source-film-database/pull/2

I'm not a maintainer, but I want this project to continue.
Let me know if you need help.

Thank you!

**Comment by malvese** [comment]
Merci ;)

I was also considering hosting it, but it looks like you already did!

**Comment by Merinorus** [comment]
Yes, this is not official but I keep maintaining it when I have some spare time, as I don't know how long the current one will be kept alive.

This was under discussion (French ;)) : https://35mm-compact.com/forum/viewtopic.php?p=1060412#p1060412

**Comment by auyongcheemeng** [comment]
the hosting could be done on [GitHub pages](https://pages.github.com/), if the website code can be adapted from its PHP form to something more client side

**Comment by Merinorus** [comment]
Yes that's a possibility, the whole database is about 2 MB. I think the db and the website should be two separate repos ideally.

**Comment by dxdatabase** [comment]
To contribute just open new issue 😸
